### PR TITLE
PaddedGraphGenerator.flow now also accepts a list of StellarGraphs

### PR DESCRIPTION
**Pull request by kieranricardo**
This PR enables PaddedGraphGenerator.flow to also accepts a list of StellarGraphs.

See: #1337

### CODE REVIEW

File: stellargraph/mapper/padded_graph_generator.py [hunk 1]
```diff
@@ -131,8 +132,13 @@ def flow(
                 f"expected batch_size to be strictly positive integer, found {batch_size}"
             )
 
+        if isinstance(graph_ilocs[0], StellarGraph):
```
Comment (huonw) [inline stellargraph/mapper/padded_graph_generator.py:135]: I think we need to do a lot more validation here; e.g. this allows a user to pass in an empty graph (#1338) or a graph without features or a heterogeneous graph.

One way to do this would be to factor out the loop (or just the loop body) from `__init__` and call it both there and here.

Given there's a variety of edge cases, I wonder if we should do the easy thing (emitting a better error) for 1.0 and do this more flexible version for 1.1?

Comment (kieranricardo) [reply stellargraph/mapper/padded_graph_generator.py:138]: good point, I had a quick go at extending existing edge case tests to handle `.floaw` and I think they're all covered

File: stellargraph/mapper/padded_graph_generator.py [hunk 2]
```diff
@@ -91,8 +95,8 @@ def flow(
         with the supplied graph indexes and targets.
 
         Args:
-            graph_ilocs (iterable): an iterable of graph indexes in self.graphs for the graphs of interest
-                (e.g., training, validation, or test set nodes).
+            graph_ilocs (iterable): an iterable of graph indexes in self.graphs or an iterable of `StellarGraph`s
```
Comment (huonw) [inline stellargraph/mapper/padded_graph_generator.py:98]: Tiny thing, but we should consider renaming this parameter to `graphs`, so the error messages from `self._check_graphs` match.

```suggestion
            graphs (iterable): an iterable of graph indexes in self.graphs or an iterable of `StellarGraph`s
```

and all the other instances of `graph_ilocs` in this file.

Comment (kieranricardo) [reply stellargraph/mapper/padded_graph_generator.py:98]: This breaks code specifying `graph_ilcs` as a kwarg but `PaddedGraphGenerator` wasn't in the last release so I think this should be fine.
